extensions/kubernetes-client/runtime-internal/src/main/java/io/quarkus/kubernetes/client/runtime/KubernetesDevServicesBuildTimeConfig.java

```diff
@@ -29,7 +29,7 @@ public class KubernetesDevServicesBuildTimeConfig {
     /**
      * The flavor to use (kind, k3s or api-only). Default to api-only.
      */
-    @ConfigItem(defaultValue = "API_ONLY")
+    @ConfigItem(defaultValue = "api-only")
```

Comment:
Should not be `api_only`?

Reply:
No. We replace `_` with `-` when dealing with enum default values in Quarkus.

Reply:
I've just tried that both values `api-only` and `api_only` are indeed working fine. 
Though, I would prefer using `api_only` which is the value present in the Flavor enum. This way we don't need to know this special handling made when using enum default values.

Reply:
We are using dashed values consistently in all Quarkus so let's keep it this way.

We decided to go with dashes for all default values.